docs/java/java-webapp.md

```diff
@@ -11,83 +11,74 @@ MetaDescription: Java web app tutorial showing how to deploy a Java web app to A
+<a class="tutorial-next-btn" href="https://azure.microsoft.com/pricing/free-trial/" target="_blank" style="background-color:#68217A">Create your free Azure account</a>
 
-## Build and deploy your web app to Azure
+The deploy process leverages the [Azure Account](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account) extension (installed along with the Azure Functions extension as an dependency) and you need to sign in with your Azure subscription. If you do not have an Azure subscription, [sign up today](https://azure.microsoft.com//free/?b=16.48) for a free 30 day account and get $200 in Azure Credits to try out any combination of Azure services.
 
-From the command prompt or terminal window you were using earlier, rebuild the JAR file using Maven if you made any changes to the `pom.xml` file. For example:
+To log into Azure, run **Azure: Sign In** from the **Command Palette** (`kb(workbench.action.showCommands)`). You can then sign into your account using the **Device Login** flow. Click on **Copy & Open** to open your default browser.
 
-```bash
-mvn clean package
-```
+![Azure sign in code](images/java-webapp/devicelogin.png)
 
-> **Note:** You need to be in the `complete` directory when running commands in the terminal.
+Paste in the access code and continue the sign in process.
 
-Deploy your web app to Azure by using Maven:
+![Azure Device Login](images/java-webapp/devicelogin2.png)
+
+Once you have signed in, you can open the command prompt or terminal window and build the project using Maven commands. This will generate a new `name.war` or `name.jar` file in the `target` directory. 
 
 ```bash
-mvn azure-webapp:deploy
+mvn clean package
 ```
+After building the project, open the `target` directory in VS Code Explorer. Right-click on the `name.war` or `name.jar` file and choose **Deploy to Web App** button, and follow the prompts to choose the Web App for your deployment.
 
-Maven will deploy your web app to Azure. If the web app does not already exist, it will be created.
-
-When your web app has been deployed, you will see a success message on the command line:
-
-![Deploy Success](images/java-webapp/deploy-success.png)
-
-By pasting the URL into a web browser, you can see the sample web app running on Azure!
-
-![Greeting cloud](images/java-webapp/greeting-cloud.png)
-
-You will also be able to manage it using the [Azure portal](https://portal.azure.com/).
-
-Your web app will be listed under **App Services**:
-
-![App Service View](images/java-webapp/app-service-view.png)
-
-The URL for your web app will be listed in the **Overview** for your web app:
+![Deploy to Web App button](images/java-webapp/deploy-webapp.png)
 
-![Overview](images/java-webapp/overview.png)
+Open the **Output** window in VS Code to view the deployment logs. Once the deployment is completed, it will print out the URL for your Web App. By pasting the URL into a web browser, you can see the web app running on Azure! 
```

Comment:
Once the deployment is completed, it will print the URL for your Web App. Click it to open in a browser, you could see the web app running on Azure!